docs/.eleventy/layouts/main.njk

```diff
@@ -5,25 +5,30 @@ description: "Snowpack is the fast, bundle-free build tool for all kinds of fron
 ---
 
 <div class="grid-body-header">
-  <svg height="64px" style="padding-left: 8px;" viewBox="0 0 640 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-      <g id="Page-1" stroke="none" stroke-width="1" fill="currentColor" fill-rule="evenodd">
-          <g id="mountain-solid" transform="translate(-1.000000, 0.000000)"fill-rule="nonzero">
-              <path d="M635.92,462.7 L347.92,14.7 C342.03,5.54 331.89,0 321,0 C310.11,0 299.97,5.54 294.08,14.7 L6.08,462.7 C-0.250773249,472.547007 -0.699487627,485.064987 4.91,495.34 C10.522069,505.612419 21.2945349,512 33,512 L609,512 C620.71,512 631.48,505.61 637.09,495.33 C642.699457,485.058495 642.250708,472.543372 635.92,462.7 Z M321,91.18 L406.39,224 L321,224 L257,288 L218.94,249.94 L321,91.18 Z" id="Shape"></path>
-          </g>
+  <svg height="64px" style="padding-left: 8px;" viewbox="0 0 640 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="Page-1" stroke="none" stroke-width="1" fill="currentColor" fill-rule="evenodd">
+      <g id="mountain-solid" transform="translate(-1.000000, 0.000000)" fill-rule="nonzero">
+        <path d="M635.92,462.7 L347.92,14.7 C342.03,5.54 331.89,0 321,0 C310.11,0 299.97,5.54 294.08,14.7 L6.08,462.7 C-0.250773249,472.547007 -0.699487627,485.064987 4.91,495.34 C10.522069,505.612419 21.2945349,512 33,512 L609,512 C620.71,512 631.48,505.61 637.09,495.33 C642.699457,485.058495 642.250708,472.543372 635.92,462.7 Z M321,91.18 L406.39,224 L321,224 L257,288 L218.94,249.94 L321,91.18 Z" id="Shape"></path>
       </g>
+    </g>
   </svg>
   <h1 class="header-snowpack">Snowpack</h1>
   <p class="header-snowpack-subtitle">
     The faster frontend build tool.
   </p>
+  <p class="header-snowpack-blurb">
+      Snowpack is a modern front-end build tool that replaces traditional bundlers like Webpack or Parcel. It's fast... really fast. Snowpack starts up in 50 milliseconds or less and and rebuilds files instantly thanks to JavaScript's native module system (<a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import">ESM</a>). When it's time to deploy, Snowpack works with your favorite tools and bundlers to create a fast, production-ready build. 
```

Comment:
to your comment on highlighting that we minimize rebuild work needed, something like this could work:

> Snowpack leverages JavaScripts native module system (ESM) to start up instantly and cache built files individually, removing all unnecessary build delays during development.

What do you think? I'm good to merge with either!